runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerOverlay.java

```diff
@@ -76,6 +85,18 @@
 		ItemID.EXPEDITIOUS_BRACELET
 	);
 
+	private ImmutableList<WidgetItem> checkInventory()
+	{
+		Query inventoryQuery = new InventoryWidgetItemQuery();
```

Comment:
Use .idEquals here

Reply:
What benefit would that bring in practice?

Reply:
It will not query all inventory items but only needed ones